pkg/actuators/machine/instances_test.go

```diff
@@ -1084,7 +1119,7 @@ func TestCorrectExistingTags(t *testing.T) {
 				mockAWSClient.EXPECT().CreateTags(gomock.Any()).Return(&ec2.CreateTagsOutput{}, nil).MinTimes(1)
 			}
 
-			err := correctExistingTags(machine, &instance, mockAWSClient)
+			err := correctExistingTags(machine, &instance, mockAWSClient, tc.userTags)
```

Comment:
This doesn't necessarily need fixing now, but just noticed that this test doesn't check what the expected tags are, we should probably add a check at some point so that we know that the code is updating the tags to the correct values